www/docs/en/dev/config_ref/index.md

```diff
@@ -45,207 +45,356 @@ passively copied into various `platforms/` subdirectories, for example:
-            <content src="index.html" />
-            <access origin="*" />
-        </widget>
-
-The following configuration elements appear in the top-level
-`config.xml` file, and are supported across all supported Cordova
-platforms:
-
-- The `<widget>` element's `id` attribute provides the app's
-  reverse-domain identifier, and the `version` its full version number
-  expressed in major/minor/patch notation.
-
-  The widget tag can also have attributes that specify alternative versions,
-  namely `versionCode` for Android, `CFBundleVersion` for iOS, and 
-  `packageVersion` for Windows. See the Additional Versioning section below 
-  for details.
-
-- The `<name>` element specifies the app's formal name, as it appears
-  on the device's home screen and within app-store interfaces.
-
-- The `<description>` and `<author>` elements specify metadata and
-  contact information that may appear within app-store listings.
-
-- The optional `<content>` element defines the app's starting
-  page in the top-level web assets directory. The default value is
-  `index.html`, which customarily appears in a project's top-level
-  `www` directory.
-
-- `<access>` elements define the set of external domains the app is
-  allowed to communicate with. The default value shown above allows
-  it to access any server. See the Domain [Whitelist Guide](../guide/appdev/whitelist/index.html) for details.
-
-- The `<preference>` tag sets various options as pairs of
-  `name`/`value` attributes. Each preference's `name` is
-  case-insensitive.  Many preferences are unique to specific
-  platforms, as listed at the top of this page. The following sections
-  detail preferences that apply to more than one platform.
-
-### Additional Versioning
+# widget
+   Root element of the config.xml document.
+    
+   Attributes(type) | Description
+   ---------------- | ------------
+   id (string) | *Required: true* <br/> Specifies the app's reverse-domain identifier, and the `version` its full version number expressed in major/minor/patch notation.
+   version (string) | *Required: true* <br/> Full version number expressed in major/minor/patch notation.
+   versionCode (string) | *Required: false* <br/> **Platforms supported: Android** <br/> Alternative version for Android.
+   CFBundleVersion (string) | *Required: false* <br/> **Platforms supported: iOS** <br/> Alternative version for iOS.  
+   packageVersion (string) | *Required: false* <br/> **Platforms supported: Windows** <br/> Alternative version for Windows.
+   packageName (string) | *Default: Cordova.Example* <br/> **Platforms supported: Windows** <br/> Package name for Windows.
+   xmlns (string) | *Required: true* <br/> namespace for the config.xml document.
+   xmlns:cdv (string) | *Required: true* <br/> namespace prefix.
+    
+   Examples:
 
-Android, iOS, and Windows support a second version string (or number) in 
-addition to the one visible in app stores:
+    <widget id="io.cordova.hellocordova">
```

Comment:
Consider providing 1 example with all the attributes. Since there are a number of required fields - most of these examples are invalid without the correct value.